src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -65,10 +65,12 @@ function LineNumberTooltip({ editor, keyModifiers }: Props) {
   const [targetNode, setTargetNode] = useState<HTMLElement | null>(null);
   const lastHoveredLineNumber = useRef<number | null>(null);
   const isMetaActive = keyModifiers.meta;
-
-  const hitCounts = useAppSelector(getHitCountsForSelectedSource);
   const source = useAppSelector(getSelectedSource);
 
+  const hitCounts = useAppSelector(state =>
+    getHitCountsForSource(state, source!.id, lastHoveredLineNumber.current || 0)
```

Comment:
:speech_balloon: The React team would probably yell at us for reading from a ref while rendering, but 🤷‍♂️